### PR TITLE
docs: update to year 2025

### PR DESCRIPTION
**Pull request by thetric**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/main/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features) (not applicable)
- [x] Documentation has been added / updated (for bug fixes / features) (not applicable)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## What is the new behavior?

Corrects the year to 2024

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/src/404-body.html [hunk 1]
```diff
@@ -42,7 +42,7 @@ <h1 class="no-toc" id="page-not-found">Resource Not Found</h1>
   <footer class="no-print">
     <aio-footer>
       <p>
-        Powered by The Community ©2015-2020.
+        Powered by The Community ©2015-2024.
```
Comment (timdeschryver) [inline projects/ngrx.io/src/404-body.html:45]:
```suggestion
        Powered by The Community ©2015-2025.
```

File: projects/ngrx.io/src/app/layout/footer/footer.component.html [hunk 2]
```diff
@@ -17,7 +17,7 @@ <h3>{{node.title}}</h3>
 </div>
 
 <p>
-    Powered by the Community ©2015-2023.
+    Powered by the Community ©2015-2024.
```
Comment (timdeschryver) [inline projects/ngrx.io/src/app/layout/footer/footer.component.html:20]:
```suggestion
    Powered by the Community ©2015-2025.
```

**Review by timdeschryver — COMMENTED** [review]
Maybe it's easier to just update it to 2025 already 😅

File: projects/ngrx.io/src/app/layout/footer/footer.component.html (hunk 2 above)
Comment (muratbeser) [reply projects/ngrx.io/src/app/layout/footer/footer.component.html:20]: " Powered by the Community ©2015 - end of time." would look cool tho.

**Review by rainerhahnekamp — APPROVED** [review]

**Comment by markostanimirovic** [comment]
Thanks @thetric!

**Pull request merged**